### ISSUE TITLE
[25.1.0] Response handle don't fires

### ISSUE THREAD

**Issue by Bardiamist** [onfido/onfido-ios-sdk#301]
### What was the expected behaviour?

Response handle should fires

### What happened instead?

Response handle don't fires

### Version info:

- Cocoapods (if applicable): 1.11.3
- Onfido SDK (Debug/Release): 25.1.0
- iOS: 15.5
- Xcode: 13.4.1
- Device/Simulator: Device
- Device/Simulator language: English (US)

### Integration configuration:

```
      [onFlow withResponseHandler:^(ONFlowResponse *response) {
        NSLog(@"It's not executes");
      }];
```

### Steps to reproduce:

After submit documents Onfido closed but response handle don't fired

Fixed by revert to 25.0.0

**Comment by gurpreet-onfido** [comment]
Hello @Bardiamist 

Thank you for bringing this to our attention. We will look into this and get back to you. 

Kind regards
Gurpreet Singh

**Comment by gurpreet-onfido** [comment]
Hello @Bardiamist The fix will be available in the release: 25.2.1. Stay tuned!

Kind regards
Gurpreet Singh

**Comment by Bardiamist** [comment]
Looks fixed in 26.0.0
